spec/support/page_objects/support/provider_show.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Support
+    class ProviderShow < PageObjects::Base
+      set_url "support/providers/{id}"
+
+      element :users_tab, ".app-tab-navigation__link", text: "Users"
+
+      def providers
```

Comment:
I don't think is needed? I couldn't see any page object using it for this PR

Reply:
Goot spot. Copy and paste fail.